pkg/services/ngalert/api/api.go

```diff
@@ -7,12 +7,12 @@ import (
 
 	"github.com/go-macaron/binding"
 
-	apimodels "github.com/grafana/alerting-api/pkg/api"
 	"github.com/grafana/grafana/pkg/api/routing"
 	"github.com/grafana/grafana/pkg/infra/log"
 	"github.com/grafana/grafana/pkg/middleware"
 	"github.com/grafana/grafana/pkg/services/datasourceproxy"
 	"github.com/grafana/grafana/pkg/services/datasources"
+	apimodels "github.com/grafana/grafana/pkg/services/ngalert/api/tooling/definitions"
```

Comment:
I would prefer the models previously defined in alerting-api to be defined under `github.com/grafana/grafana/pkg/services/ngalert/models`.
This doesn't necessarily need to happen in this iteration but we could move them in a following iteration.